samples/apps/logging/logging.cpp

```diff
@@ -250,7 +253,7 @@ namespace loggingapp
             std::move(error_reason));
         }
 
-        auto records_handle = ctx.tx.template rw<RecordsTable>("records");
+        auto records_handle = ctx.tx.template rw<RecordsTable>(PRIVATE_RECORDS);
```

Comment:
I feel like we used to have 80/20 `tables`/`maps`, and we're about 40/60 now? I have a mild inclination to go all `maps`, but what does everyone else think?

Reply:
I'm wary of `map` because its overloaded (verb vs noun, type vs operation, existing std type), but I think since we've settled on `Map` for the core type you're right we should probably consolidate on `map` for the specialized types and variables.

Reply:
So I've renamed `Table` to `Map` for these types in `logging.cpp`, but going to punt on a wider rename. Renaming `NetworkTables` and `ccf::Tables` is a breaking change, so deserves a bit more thought (maybe I roll it into the statically-named-maps PR I was working on, which touches most of these references anyway...)

Reply:
I think that all other things being equal I'd rather be consistent with what we've decided too, but I do agree that we have to consider if the break is worth the improvement.